src/applications/vaos/components/calendar/CalendarSelectedIndicatorBubbles.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function CalendarSelectedIndicatorBubbles({
+  date,
+  selectedDates,
+  fieldName,
+}) {
+  return (
+    <div className="vaos-calendar__indicator-bubbles-container">
+      {[...selectedDates]
+        .filter(currentDate => currentDate.date === date)
```

Comment:
I might throw in a `.map(selected => selected[fieldName])` after the filter. I think that makes the following code a little clearer, because you can remove a couple instances `d[fieldName]` code, which isn't very clear.